src/containers/Notifier.tsx

```diff
@@ -0,0 +1,21 @@
+import { clearNotification } from '@actions'
+import { RootState } from '@store'
+import * as React from 'react'
+import { connect } from 'react-redux'
+
+const mapStateToProps = ({ notifier }: RootState) => ({ ...notifier })
+
+const mapDispatchToProps = { clearNotification }
+
+type Props = typeof mapDispatchToProps & ReturnType<typeof mapStateToProps>
+
+const NotifierComponent: React.SFC<Props> = (props) => {
+  return (
```

Comment:
Can get rid of the return by just having the parentheses immediately after the arrow